### PR TITLE
update version

### PR DESCRIPTION
**Pull request by VaShu**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "ekreative/redmine_login",
+    "name": "vashu/redmine_login",
```
Comment (mcfedr) [inline composer.json]: no

File: composer.json [hunk 2]
```diff
@@ -14,12 +14,12 @@
         }
     },
     "require": {
-        "php": ">=5.5",
-        "symfony/symfony": "^2.8.0",
-        "guzzlehttp/guzzle": "^6.1.0",
-        "mcfedr/json-form": "^3.1.0",
+        "php": ">=5.6",
+        "symfony/symfony": "^3.0.0",
```
Comment (mcfedr) [inline composer.json]: i think it would be better to put `"^2.8|^3.0"`

**Comment by VaShu** [comment]
fixed by comments

File: composer.json [hunk 3]
```diff
@@ -14,12 +14,12 @@
         }
     },
     "require": {
-        "php": ">=5.5",
-        "symfony/symfony": "^2.8.0",
-        "guzzlehttp/guzzle": "^6.1.0",
-        "mcfedr/json-form": "^3.1.0",
+        "php": ">=5.6",
```
Comment (mcfedr) [inline composer.json:17]: Is there a need for 5.6?

**Pull request merged**